ui/apps/platform/src/messages/vulnMgmt.messages.ts

```diff
@@ -34,3 +34,14 @@ export const imageScanMessages = {
         body: 'The scan results are not certified, as the base RHEL image is out of scope for certification. Please consider updating the base image.',
     },
 };
+
+export const nodeScanMessages = {
+    missingScanData: {
+        header: 'Failed to get scan data.',
+        body: 'Failed to get scan data. There may have been an error communicating with the integrated node scanner.',
+    },
+    osUnsupported: {
+        header: 'Node OS unsupported.',
+        body: 'The node’s OS is not supported at this time. Please contact ???',
```

Comment:
TODO: replace with something

Reply:
If you don't get a prompt on this, I'd go ahead and replace it with something generic, like "Please contact your infrastructure manager."

I'd rather see a later 1-line PR to update than risk merge conflicts by keeping all this code out of the main line of development.